java/src/jmri/jmrix/loconet/LnPowerManager.java

```diff
@@ -179,8 +180,22 @@ public void run() {
             msg.setOpCode(LnConstants.OPC_RQ_SL_DATA);
             msg.setElement(1, 0);
             msg.setElement(2, 0);
-            tc.sendLocoNetMessage(msg);
+            try {
```

Comment:
This try/catch/try/catch gives a maximum number of two attempts, and will suit any system where 1.5 seconds "is sufficient".

I had assumed a "try, with sufficient delay between tries, until success" for this type of protection against delayed traffic controller initialization...

Reply:
Thanks. That calls for [try-catch-finally](http://tutorials.jenkov.com/java-exception-handling/basic-try-catch-finally.html) or ``while``. Will try (!) to construct one.